pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.black]
 line-length = 88
-target-version = ['py36', 'py37', 'py38']
+target-version = ['py36', 'py37', 'py38', 'py39']
```

Comment:
We can probably stop targeting py36 and py37 since Python 3.7 is end of life

Reply:
yep, just left it in without realizing 3.7 is EOL, thanks, I'll change that now too.